test/test-exporter.js

```diff
@@ -28,29 +28,26 @@ module.exports = (repo) => {
     it('ensure hash inputs are sanitized', (done) => {
       const hash = 'QmQmZQxSKQppbsWfVzBvg59Cn3DKtsNVQ94bjAxg2h3Lb8'
       const mhBuf = new Buffer(bs58.decode(hash))
+      const cid = new CID(hash)
 
-      pull(
-        ipldResolver.getStream(new CID(hash)),
```

Comment:
It's only available internally, to make the API match straight up what is the dag API.